### ISSUE TITLE
indentation: false positive in when pseudo-class used in less

### ISSUE THREAD

**Issue by danpoltawski** [stylelint/stylelint#2087]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

A bug where an indention problem is reported incorrectly when the selector uses  the `:not()` pseudo-class 

> Which rule, if any, is this issue related to?

`indentation`

> What CSS is needed to reproduce this issue?

```css
.foo:not(.unresponsive) .one,
.foo:not(.unresponsive) .two{
    padding: 0;
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "indentation": 4
  }
}
```

> Which version of stylelint are you using?

`7.6.0` (but found in earlier versions)

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI with `stylelint test.less`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Yes, the problem only occurs when syntax is set to less.

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

The following warning was flagged:

```shell
test.less
 2:1  ✖  Expected indentation of 4 spaces   indentation
```

Note that the  same issue doesn't happen when using plain css syntax:
```
$ cat test.css 
.foo:not(.unresponsive) .one,
.foo:not(.unresponsive) .two{
    padding: 0;
}

✔
09:33 $ stylelint test.css 
✔ 
09:33 $ stylelint -s less test.css 

test.css
 2:1  ✖  Expected indentation of 4 spaces   indentation

✘-2
```

**Comment by evilebottnawi** [comment]
@danpoltawski seem duplicate https://github.com/stylelint/stylelint/issues/1741

**Comment by danpoltawski** [comment]
Apologies for the duplicate, closing

**Comment by jeddy3** [comment]
@danpoltawski No worries :)

FYI, that bug is outstanding as the [PostCSS Less parser](https://github.com/webschik/postcss-less/) currently lacks contributors. So, please consider lending your support to that repo if you'd like to see the issue resolved.
